docker-plugin/src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/help-volumesString.html

```diff
@@ -1,3 +1,10 @@
 <div>
-    A space separated list of host volume mounts, e.g. /host/path:/container/path:ro
+    A space separated list of host volume mounts : &lt;host/path&gt;[&lt;container/path&gt;[&lt;:mode&gt;]]
+    <ul>
+        <li>container/path create empty  volume container/path</li>
+        <li>host/path:container/path will mount host/path from host to container's container/path, read/write</li>
+        <li>host/path:container/path:rw will mount host/path from host to container's container/path, read/write</li>
+        <li>host/path:container/path:ro will mount host/path from host to container's container/path, read-only</li>
+    </ul>
+    Note: when provided, an invalid mode will mount the volume in read-only
```

Comment:
Better don't mount at all imho. Either it will be not obvious. Also all input data must be checked with FormValidation.